docs/source/includes/tute_data.rst

```diff
@@ -8,5 +8,5 @@ See `Tute's documentation`_ for more details about:
 Google Cloud Platform data locations
 ------------------------------------
 
-* Google Cloud Storage folder `gs://tute_db <https://console.developers.google.com/storage/tute_db>`_
+* Google Cloud Storage folder `gs://tute_db <https://console.cloud.google.com/storage/tute_db>`_
 * Google BigQuery Dataset ID `silver-wall-555:TuteTable.hg19 <https://bigquery.cloud.google.com/table/silver-wall-555:TuteTable.hg19>`_
```

Comment:
Hi Matt,

I see the same message for both.  So below are the steps of what I perform and what I see:

1) Via gsutil:

```Bash
$ gsutil ls gs://tute_db | head -n 3
gs://tute_db/hg19allsnp.split1.ggapidb.txt.gz
gs://tute_db/hg19allsnp.split10.ggapidb.txt.gz
gs://tute_db/hg19allsnp.split100.ggapidb.txt.gz
$
$ gsutil cat gs://tute_db/hg19allsnp.split100.ggapidb.txt.gz | gunzip - | head
Failure: The account for bucket &quot;tute_db&quot; has been disabled..

gzip: stdin: unexpected end of file
$
```
2) Next I perform the same thing, by trying it through the browser and clicking on the following link:

https://console.cloud.google.com/m/cloudstorage/b/tute_db/o/hg19allsnp.split100.ggapidb.txt.gz

The response I get from the browser is the following:

```
The account for bucket "tute_db" has been disabled.
```
It seemed to have worked before, since I performed an analysis through R on the files, and posted it at the following location a while back:

https://groups.google.com/forum/#!topic/tute-genomics/wW5ubyPDV-Y

Thank you for helping me out.

Paul